src/components/base/Base.js

```diff
@@ -668,11 +668,33 @@ export class BaseComponent {
     let addIcon = this.ce('span', {
       class: 'glyphicon glyphicon-plus'
     });
+
     addButton.appendChild(addIcon);
     addButton.appendChild(this.text(this.component.addAnother || ' Add Another'));
     return addButton;
   }
 
+  /**
+   * Adds a new icon button to  new rows for multiple input elements.
+   * @returns {HTMLElement} - The "+" button html element.
+   */
+  addIconButton(){
```

Comment:
This is VERY similar to the ```addButton``` method. Why did you create a new method here. You could very easily pass in a parameter to the ```addButton``` method to not add the text piece of it if that variable is set, and it would make the code much cleaner.